examples/Billing/AddBillingSetup.cs

```diff
@@ -61,18 +63,15 @@ public static void Main(string[] args)
         /// <summary>
         /// Returns a description about the code example.
         /// </summary>
-        public override string Description
-        {
-            get
-            {
-                return "This example creates a billing setup for a customer. A billing setup is " +
-                       "a link between payment account and customer. The new billing setup can " +
-                       "either reuse an existing payments account, or create a new payments " +
-                       "account with a given payments profile.\n" +
-                       "In the case of consolidated billing, a payments account is linked to the " +
-                       "manager account and is linked to a customer account via a billing setup.";
-            }
-        }
+        public override string Description =>
+            "This example creates a billing setup for a customer. A billing setup is " +
+            "a link between payment account and customer. The new billing setup can " +
```

Comment:
"payment**s** account", "and **a** customer"

Reply:
Fixed.